resources/ory/charts/oathkeeper/values.yaml

```diff
@@ -1,109 +1,112 @@
-# Mode for oathkeeper controller
-# Two possible modes are: controller or sidecar
+# -- Mode for oathkeeper controller
+# -- Two possible modes are: controller or sidecar
 global:
   ory:
     oathkeeper:
       maester:
         mode: controller
 
-# Number of ORY Oathkeeper members
+# -- Number of ORY Oathkeeper members
 replicaCount: 1
 
 image:
-  # Image pull policy
+  # Kyma controlled
+  # -- Image pull policy
   pullPolicy: IfNotPresent
 
+# -- Options for the sidecar
 sidecar:
   port:
     metrics: 8080
 
-# Image pull secrets
+# -- Image pull secrets
 imagePullSecrets: []
-# Chart name override
+# -- Chart name override
 nameOverride: ""
-# Full chart name override
+# -- Full chart name override
 fullnameOverride: ""
 
-# If enabled, a demo deployment with exemplary access rules
-# and JSON Web Key Secrets will be generated.
+# -- If enabled, a demo deployment with exemplary access rules and JSON Web Key Secrets will be generated.
 demo: false
 
-# Configures the Kubernetes service
+# -- Configures the Kubernetes service
 service:
-  # Configures the Kubernetes service for the proxy port.
+  # -- Configures the Kubernetes service for the proxy port.
   proxy:
-    # En-/disable the service
+    # -- En-/disable the service
     enabled: true
-    # The service type
+    # -- The service type
     type: ClusterIP
-    # The service port
+    # -- The service port
     port: 4455
-    # If you do want to specify annotations, uncomment the following
+    # -- The service port name. Useful to set a custom service port name if it must follow a scheme (e.g. Istio)
+    name: http
+    # -- If you do want to specify annotations, uncomment the following
     # lines, adjust them as necessary, and remove the curly braces after 'annotations:'.
-    annotations: {}
     # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
+    annotations: {}
+    # -- If you do want to specify additional labels, uncomment the following lines, adjust them as necessary, and remove the curly braces after 'labels:'.
+    # e.g.  app: oathkeeper
     labels: {}
-    #      If you do want to specify additional labels, uncomment the following
-    #      lines, adjust them as necessary, and remove the curly braces after 'labels:'.
-    #      e.g.  app: oathkeeper
 
-  # Configures the Kubernetes service for the api port.
+  # -- Configures the Kubernetes service for the api port.
   api:
```

Comment:
I think we miss `service.api.name` value here

Reply:
service.api.name = "" added with a comment